### ISSUE TITLE
Making request with brewtils v3 SystemClient against v2 beer-garden results in "Unable to send a request with no system namespace" error

### ISSUE THREAD

**Issue by astephon88** [beer-garden/beer-garden#827]
After updating to brewtils v3, requests sent with SystemClient to a v2 beergarden are failing with a "Unable to send a request with no system namespace" error. Passing an empty string as the system_namespace parameter when the SystemClient instance is created to evalutate to the local garden’s default namespace as suggested in the brewtils documentation did not resolve the issue. 

The client is initialized as follows:
```python
    client = SystemClient(
        host=host,
        port=port,
        system_name=system_name,
        system_namespace="",
        version_constraint=version,
        default_instance=instance_name,
        ssl_enabled=ssl_enabled,
        url_prefix=prefix,
        ca_verify=ca_verify,
        ca_cert=_get_ca_path(),
        client_cert=client_cert,
        client_key=client_key,
        blocking=False,
    )

    client.load_bg_system()
```

And requests are made via:
`getattr(client, command)(**parameters)`

I am currently able to work around the bug by manually overriding the system_namespace when calling the system command
`getattr(client, command)(_system_namespace="", **parameters)`

**Comment by hazmat345** [comment]
Hi @astephon88 - thanks for letting us know about this. I've opened a PR to address.

With this fix the workaround of manually passing `system_namespace=""` won't be necessary - that's actually the default. The issue was that we use everything you pass to the `SystemClient` init to search for the "real" system that exists on the Beergarden server, and then use *that* information when creating requests.

When you're running against a v2 Beergarden the namespace that's returned will always be `None`, so that's what we use.

This is all "correct," right up until the part where we explicitly disallow creating Requests with no namespace (in the same way we don't allow Requests with no command name, version, etc.) I just removed that check and things seem to work now.
